source/nodejs/README.md

```diff
@@ -13,13 +13,16 @@ This directory contains all the JS packages for adaptive cards:
 Removed JS packages:
 * `adaptivecards-fabric` - A set of Fabric UI extensions for the Adaptive Cards JS renderer. This package has been deprecated. Please migrate to [`adaptivecards-fluentui`](https://www.npmjs.com/package/adaptivecards-fluentui). The source code is available in [21.09](https://github.com/microsoft/AdaptiveCards/releases/tag/21.09) or before releases.
 
+## Authenticate with npm registry
+
+1. `vsts-npm-auth -config .npmrc` in `source/nodejs`
+
 ## Bootstrap the repo
 
```

Comment:
their requirements are a _bit_ different because they're actually running their code server-side inside node. we only use node as a toolchain to generate JS to run in the browser. that said, it's probably worth a shout-out...